src/components/orgs/Coupon/Container.tsx

```diff
@@ -220,6 +220,22 @@ const CouponContainer: React.FC<CouponContainerProps> = ({
 
   const handleActiveCoupon = async () => {
     if (session?.user) {
+      if (isDateValid(session.user.subscriptionEndDate) === false) {
```

Comment:
mesma coisa pra funcao de isPremiumUser

Reply:
criei o util no pr : https://github.com/Agility-Creative-Solutions/ez2live-app/pull/159
Assim que for mergeado irei corrigir aqui. Só repois irei colocar como corrigido